### PR TITLE
Feature/147 AKS script documentation

### LINKED ISSUE #147: AKS: Automation documentation
## Description
- [ ] Document script usage

- References #105

### PR DESCRIPTION
**Pull request by kforeverisback**
# Type of PR

- [X] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Add AKS script documentation.

## Validation

- [ ] Unit tests updated and ran successfully
- [X] Update documentation or issue referenced above

## Issues Closed or Referenced
Should be closed after PR #161 

- Closes #147 
- References #105

### CODE REVIEW

File: IaC/scripts/create-cluster-env.bash [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+        shift 2
+        ;;
+
+        -r|--dns-rg)
+        Ngsa_DNS_RG=$val
+        shift 2
+        ;;
+
+        -o|--smoke-it)
+        Ngsa_Smoke=yes
```
Comment (jofultz) [inline IaC/scripts/create-cluster-env.bash:96]: Should we calls this Ngsa_Loderunner or Ngsa_l8r?

File: IaC/scripts/create-cluster-env.bash [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+    # Check if DNS Zone exists
+    if ! az network dns zone show --name $Ngsa_Domain_Name -g $Ngsa_DNS_RG -o table ${subs_arg}; then
+        # If not found, create the DNS Zone.
+        aze network dns zone create -g $Ngsa_DNS_RG -n $Ngsa_Domain_Name -o table ${subs_arg}
+    fi
+    # Add DNS A record for the Istio ingress gateway.
+    aze network dns record-set a add-record -g $Ngsa_DNS_RG -z $Ngsa_Domain_Name -n $Ngsa_Name -a $INGRESS_PIP -o table ${subs_arg}
+
+    # Show the Azure nameservers for your DNS Zone.
+    aze network dns zone show -n $Ngsa_Domain_Name -g $Ngsa_DNS_RG --query nameServers -o tsv ${subs_arg}
```
Comment (jofultz) [inline IaC/scripts/create-cluster-env.bash:338]: Lines 326, 332, 335, and 338, should those be "az network ..." instead of "aze network ..."?

**Review by jofultz — CHANGES_REQUESTED** [review]
Take a look especially at the "aze" comment.

**Review by sivamu — COMMENTED** [review]
It'd be nice to isolate the documentation changes to this PR, since this closes the documentation task only. If the branch is cloned from the `feature/105-aks-script` (which would make sense why all the script changes are showing up), then removing the script changes can be difficult. Otherwise, it'll be a bit tricky to sync the script changes per every review comment.

File: IaC/scripts/create-cluster-env.bash (hunk 1 above)
Comment (kforeverisback) [reply IaC/scripts/create-cluster-env.bash:96]: Yes, I'm changing it in the AKS-Script PR. Will change it here afterwards

**Review by kforeverisback — COMMENTED** [review]
> It'd be nice to isolate the documentation changes to this PR, since this closes the documentation task only. If the branch is cloned from the `feature/105-aks-script` (which would make sense why all the script changes are showing up), then removing the script changes can be difficult. Otherwise, it'll be a bit tricky to sync the script changes per every review comment.

I branched this off `feature/105-aks-script` so that, the links in the .md file doesn't break the build. If the readme was a standalone one it might fail the build.

> Take a look especially at the "aze" comment.
@jofultz There is a separate PR going on for the Script #161. The PR is specified in the description.

**Review by jofultz — APPROVED** [review]

**Pull request merged**